test-ciphers.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+console_url=$(oc get console.config.openshift.io cluster --template '{{.status.consoleURL}}')
+if [ -z "${console_url}" ]
+then
+  echo "no console_url, are you connected to a cluster?"
+  exit 1
+fi
+
+# need to format the console_url for s_client
+console_url_without_http=${console_url#"https://"}
+console_url_without_http_with_port="${console_url_without_http}:443"
+SERVER="${console_url_without_http_with_port}"
```

Comment:
Avoid two different variables with the same value. just use `console_url_without_http_with_port` below

Reply:
Definitely.